views/homepage.handlebars

```diff
@@ -1,15 +1,16 @@
-<ol class="topic-list">
-  {{#each topic}}
-  <li>
-    <article class="topic">
-      <div class="title">
-        <a href="{{post_url}}" target="_blank">{{title}}</a>
-        <span>({{topic_url}})</span>
-      </div>
-      <div class="meta">
-        <a href="/topic/{{id}}">{{message.length}} messages(s)</a>
-      </div>
-    </article>
-  </li>
-  {{/each}}
-</ol>
\ No newline at end of file
+<div class="homepage-section">
+<h1>Welcome, From Your Friend Teach Bot!</h1>
+<p>I am a learning bot designed to help you with your programming needs. My goal is to help you improve your programming knowledge and answer your most desired developer questions. Above you will find the Signup or the Login link to get started. Enjoy!</p>
+</div>
+<br>
+<div class="sub-section">
+  <h2>What We Can Help You With</h2>
+</div>
+
+<div class="row">
+  <div class="column" style="background-color:rgb(192, 200, 191);">Javascript</div>
+  <div class="column" style="background-color:rgb(192, 200, 191);">HTML</div>
+  <div class="column" style="background-color:rgb(192, 200, 191);">CSS</div>
+</div>
+
```

Comment:
There need to be more than this, describe how to use the website

Reply:
added more but not finalized